src/pages/workspaces/workspace/Dashboard.js

```diff
@@ -536,6 +552,7 @@ const WorkspaceDashboard = _.flow(
       ])
     ]),
     div({ style: Style.dashboard.rightBox }, [
+      _.some(isV1Artifact(workspace?.workspace), analysesData?.persistentDisks) && h(UnboundDiskNotification),
```

Comment:
Can `isV1Artifact` actually handle undefined as the argument?

Reply:
No, but it's never invoked with an undefined `workspace` argument.

Reply:
Then remove the `?` usage. That's indicating that it is expected that `workspace` could be undefined, and if that is the case, `isV1Artifact` needs to be able to handle `undefined`.

Reply:
Ah - misunderstood how javascript destructuring worked. I thought it could be partially invoked when `workspace` was undefined.

Reply:
I'm leaving the chain in the other use as the `workspace` object could be undefined